Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM node:18-bullseye-slim AS builder
 WORKDIR /app
 
 COPY . .
+COPY ./config/config.template.ts ./config/index.ts
```

Comment:
Where does the variable substitution for this happen? Just to be sure these changes don't accidentally set `WEBAUTHN_RP_ID` to `"localhost"` in prod deployments too, for example.

Reply:
Thanks for the notice. The substitution is done by mounting a `config/index.js` file which contains the "production" configuration. The reason why I added this line is because without it the building phase would fail because all files are importing the `config` object from the `config/index.ts` file.